### PR TITLE
Übersetzung hinzugefügt

### PR DESCRIPTION
**Pull request by Justman10000**

### CODE REVIEW

File: locale/justoverclock-flarum-ext-realtimecode [hunk 1]
```diff
@@ -0,0 +1,5 @@
+  forum:
```
Comment (Kakifrucht) [inline locale/justoverclock-flarum-ext-realtimecode:1]: First line must be the original first line (file name must be first line of original file + '.yml' suffix.
In this case flarum-ext-realtimecode.yml.

Comment (Kakifrucht) [inline locale/justoverclock-flarum-ext-realtimecode:1]: First line must be the original first line (file name must be first line of original file + '.yml' suffix.
In this case flarum-ext-realtimecode.yml.

File: locale/justoverclock-flarum-ext-realtimecode [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    title: HTML Editor
+    htmleditor: HTML Editor
+    description: Dieser HTML Editor unterstützt HTML/CSS code, Schreibe einfach deinen Code hier rein und schaue dir die Echtzeit-Vorschau an..
```
Comment (Kakifrucht) [inline locale/justoverclock-flarum-ext-realtimecode:4]: HTML/CSS Code 
"schreibe" behind the comma lowercase, original tl doesn't include period at the end, please remove them.

**Review by Kakifrucht — CHANGES_REQUESTED** [review]

File: locale/justoverclock-flarum-ext-realtimecode (hunk 1 above)
Comment (Justman10000) [reply locale/justoverclock-flarum-ext-realtimecode:1]: Hab´

**Comment by Kakifrucht** [comment]
Filename should be "flarum-ext-realtimecode.yml" (just as the first line of the original translation files indicate).

**Comment by Kakifrucht** [comment]
Not quite!

Look at the original translation file. We keep the exact same structure, only changing the actual translations in those files, not the translation keys. The *filename* (not the first line) must contain the .yml ending and must be the same name as the first line of the original translation file. So in this case it would be flarum-ext-realtimecode.yml as filename, and ``flarum-ext-realtimecode:`` in the first line of the translation file.

**Comment by Justman10000** [comment]
> 
> 
> Not quite!
> 
> Look at the original translation file. We keep the exact same structure, only changing the actual translations in those files, not the translation keys. The _filename_ (not the first line) must contain the .yml ending and must be the same name as the first line of the original translation file. So in this case it would be flarum-ext-realtimecode.yml as filename, and `flarum-ext-realtimecode:` in the first line of the translation file.

Didn't I have that before?

**Comment by Kakifrucht** [comment]
No, it says "justoverclock-flarum-ext-realtimecode" but it should be "flarum-ext-realtimecode"

**Comment by Kakifrucht** [comment]
Still not quite there yet. I decided to merge your commits and fix them myself (https://github.com/flarum-lang/german/commit/d5fe8284df8a233bf8e451f3ba1fa4774a0424cc).

**Pull request merged**